### ISSUE TITLE
New submissions for Mon, 22 Feb 21

### ISSUE THREAD

**Issue by kobiso** [kobiso/daily-arxiv-noti#279]
## Keyword: metric learning
There is no result 
## Keyword: image retrieval
There is no result 
## Keyword: face recognition
There is no result 
## Keyword: imagenet
There is no result 
### Improving DeepFake Detection Using Dynamic Face Augmentation
 - **Authors:** Sowmen Das, Arup Datta, Md. Saiful Islam, Md. Ruhul Amin
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Artificial Intelligence (cs.AI); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2102.09603
 - **Pdf link:** https://arxiv.org/pdf/2102.09603
 - **Abstract**
 The creation of altered and manipulated faces has become more common due to the improvement of DeepFake generation methods. Simultaneously, we have seen detection models' development for differentiating between a manipulated and original face from image or video content. We have observed that most publicly available DeepFake detection datasets have limited variations, where a single face is used in many videos, resulting in an oversampled training dataset. Due to this, deep neural networks tend to overfit to the facial features instead of learning to detect manipulation features of DeepFake content. As a result, most detection architectures perform poorly when tested on unseen data. In this paper, we provide a quantitative analysis to investigate this problem and present a solution to prevent model overfitting due to the high volume of samples generated from a small number of actors. We introduce Face-Cutout, a data augmentation method for training Convolutional Neural Networks (CNN), to improve DeepFake detection. In this method, training images with various occlusions are dynamically generated using face landmark information irrespective of orientation. Unlike other general-purpose augmentation methods, it focuses on the facial information that is crucial for DeepFake detection. Our method achieves a reduction in LogLoss of 15.2% to 35.3% on different datasets, compared to other occlusion-based augmentation techniques. We show that Face-Cutout can be easily integrated with any CNN-based recognition model and improve detection performance.
### Multilingual Augmenter: The Model Chooses
 - **Authors:** Matthew Ciolino, David Noever, Josh Kalin
 - **Subjects:** Computation and Language (cs.CL)
 - **Arxiv link:** https://arxiv.org/abs/2102.09708
 - **Pdf link:** https://arxiv.org/pdf/2102.09708
 - **Abstract**
 Natural Language Processing (NLP) relies heavily on training data. Transformers, as they have gotten bigger, have required massive amounts of training data. To satisfy this requirement, text augmentation should be looked at as a way to expand your current dataset and to generalize your models. One text augmentation we will look at is translation augmentation. We take an English sentence and translate it to another language before translating it back to English. In this paper, we look at the effect of 108 different language back translations on various metrics and text embeddings.
### Unit selection synthesis based data augmentation for fixed phrase  speaker verification
 - **Authors:** Houjun Huang, Xu Xiang, Fei Zhao, Shuai Wang, Yanmin Qian
 - **Subjects:** Sound (cs.SD); Audio and Speech Processing (eess.AS)
 - **Arxiv link:** https://arxiv.org/abs/2102.09817
 - **Pdf link:** https://arxiv.org/pdf/2102.09817
 - **Abstract**
 Data augmentation is commonly used to help build a robust speaker verification system, especially in limited-resource case. However, conventional data augmentation methods usually focus on the diversity of acoustic environment, leaving the lexicon variation neglected. For text dependent speaker verification tasks, it's well-known that preparing training data with the target transcript is the most effectual approach to build a well-performing system, however collecting such data is time-consuming and expensive. In this work, we propose a unit selection synthesis based data augmentation method to leverage the abundant text-independent data resources. In this approach text-independent speeches of each speaker are firstly broke up to speech segments each contains one phone unit. Then segments that contain phonetics in the target transcript are selected to produce a speech with the target transcript by concatenating them in turn. Experiments are carried out on the AISHELL Speaker Verification Challenge 2019 database, the results and analysis shows that our proposed method can boost the system performance significantly.
### AISPEECH-SJTU accent identification system for the Accented English  Speech Recognition Challenge
 - **Authors:** Houjun Huang, Xu Xiang, Yexin Yang, Rao Ma, Yanmin Qian
 - **Subjects:** Sound (cs.SD); Audio and Speech Processing (eess.AS)
 - **Arxiv link:** https://arxiv.org/abs/2102.09828
 - **Pdf link:** https://arxiv.org/pdf/2102.09828
 - **Abstract**
 This paper describes the AISpeech-SJTU system for the accent identification track of the Interspeech-2020 Accented English Speech Recognition Challenge. In this challenge track, only 160-hour accented English data collected from 8 countries and the auxiliary Librispeech dataset are provided for training. To build an accurate and robust accent identification system, we explore the whole system pipeline in detail. First, we introduce the ASR based phone posteriorgram (PPG) feature to accent identification and verify its efficacy. Then, a novel TTS based approach is carefully designed to augment the very limited accent training data for the first time. Finally, we propose the test time augmentation and embedding fusion schemes to further improve the system performance. Our final system is ranked first in the challenge and outperforms all the other participants by a large margin. The submitted system achieves 83.63\% average accuracy on the challenge evaluation data, ahead of the others by more than 10\% in absolute terms.
### CatNet: music source separation system with mix-audio augmentation
 - **Authors:** Xuchen Song, Qiuqiang Kong, Xingjian Du, Yuxuan Wang
 - **Subjects:** Sound (cs.SD); Audio and Speech Processing (eess.AS)
 - **Arxiv link:** https://arxiv.org/abs/2102.09966
 - **Pdf link:** https://arxiv.org/pdf/2102.09966
 - **Abstract**
 Music source separation (MSS) is the task of separating a music piece into individual sources, such as vocals and accompaniment. Recently, neural network based methods have been applied to address the MSS problem, and can be categorized into spectrogram and time-domain based methods. However, there is a lack of research of using complementary information of spectrogram and time-domain inputs for MSS. In this article, we propose a CatNet framework that concatenates a UNet separation branch using spectrogram as input and a WavUNet separation branch using time-domain waveform as input for MSS. We propose an end-to-end and fully differentiable system that incorporate spectrogram calculation into CatNet. In addition, we propose a novel mix-audio data augmentation method that randomly mix audio segments from the same source as augmented audio segments for training. Our proposed CatNet MSS system achieves a state-of-the-art vocals separation source distortion ratio (SDR) of 7.54 dB, outperforming MMDenseNet of 6.57 dB evaluated on the MUSDB18 dataset.
### MolCLR: Molecular Contrastive Learning of Representations via Graph  Neural Networks
 - **Authors:** Yuyang Wang, Jianren Wang, Zhonglin Cao, Amir Barati Farimani
 - **Subjects:** Machine Learning (cs.LG); Chemical Physics (physics.chem-ph)
 - **Arxiv link:** https://arxiv.org/abs/2102.10056
 - **Pdf link:** https://arxiv.org/pdf/2102.10056
 - **Abstract**
 Molecular machine learning bears promise for efficient molecule property prediction and drug discovery. However, due to the limited labeled data and the giant chemical space, machine learning models trained via supervised learning perform poorly in generalization. This greatly limits the applications of machine learning methods for molecular design and discovery. In this work, we present MolCLR: Molecular Contrastive Learning of Representations via Graph Neural Networks (GNNs), a self-supervised learning framework for large unlabeled molecule datasets. Specifically, we first build a molecular graph, where each node represents an atom and each edge represents a chemical bond. A GNN is then used to encode the molecule graph. We propose three novel molecule graph augmentations: atom masking, bond deletion, and subgraph removal. A contrastive estimator is utilized to maximize the agreement of different graph augmentations from the same molecule. Experiments show that molecule representations learned by MolCLR can be transferred to multiple downstream molecular property prediction tasks. Our method thus achieves state-of-the-art performance on many challenging datasets. We also prove the efficiency of our proposed molecule graph augmentations on supervised molecular classification tasks.
### Mine Your Own vieW: Self-Supervised Learning Through Across-Sample  Prediction
 - **Authors:** Mehdi Azabou, Mohammad Gheshlaghi Azar, Ran Liu, Chi-Heng Lin, Erik C. Johnson, Kiran Bhaskaran-Nair, Max Dabagia, Keith B. Hengen, William Gray-Roncal, Michal Valko, Eva L. Dyer
 - **Subjects:** Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2102.10106
 - **Pdf link:** https://arxiv.org/pdf/2102.10106
 - **Abstract**
 State-of-the-art methods for self-supervised learning (SSL) build representations by maximizing the similarity between different augmented "views" of a sample. Because these approaches try to match views of the same sample, they can be too myopic and fail to produce meaningful results when augmentations are not sufficiently rich. This motivates the use of the dataset itself to find similar, yet distinct, samples to serve as views for one another. In this paper, we introduce Mine Your Own vieW (MYOW), a new approach for building across-sample prediction into SSL. The idea behind our approach is to actively mine views, finding samples that are close in the representation space of the network, and then predict, from one sample's latent representation, the representation of a nearby sample. In addition to showing the promise of MYOW on standard datasets used in computer vision, we highlight the power of this idea in a novel application in neuroscience where rich augmentations are not already established. When applied to neural datasets, MYOW outperforms other self-supervised approaches in all examples (in some cases by more than 10%), and surpasses the supervised baseline for most datasets. By learning to predict the latent representation of similar samples, we show that it is possible to learn good representations in new domains where augmentations are still limited.
### An Empirical Study on Measuring the Similarity of Sentential Arguments  with Language Model Domain Adaptation
 - **Authors:** ChaeHun Park, Sangwoo Seo
 - **Subjects:** Computation and Language (cs.CL)
 - **Arxiv link:** https://arxiv.org/abs/2102.09786
 - **Pdf link:** https://arxiv.org/pdf/2102.09786
 - **Abstract**
 Measuring the similarity between two different sentential arguments is an important task in argument mining. However, one of the challenges in this field is that the dataset must be annotated using expertise in a variety of topics, making supervised learning with labeled data expensive. In this paper, we investigated whether this problem could be alleviated through transfer learning. We first adapted a pretrained language model to a domain of interest using self-supervised learning. Then, we fine-tuned the model to a task of measuring the similarity between sentences taken from different domains. Our approach improves a correlation with human-annotated similarity scores compared to competitive baseline models on the Argument Facet Similarity dataset in an unsupervised setting. Moreover, we achieve comparable performance to a fully supervised baseline model by using only about 60% of the labeled data samples. We believe that our work suggests the possibility of a generalized argument clustering model for various argumentative topics.
### ISCL: Interdependent Self-Cooperative Learning for Unpaired Image  Denoising
 - **Authors:** Kanggeun Lee, Won-Ki Jeong
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG); Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2102.09858
 - **Pdf link:** https://arxiv.org/pdf/2102.09858
 - **Abstract**
 With the advent of advances in self-supervised learning, paired clean-noisy data are no longer required in deep learning-based image denoising. However, existing blind denoising methods still require the assumption with regard to noise characteristics, such as zero-mean noise distribution and pixel-wise noise-signal independence; this hinders wide adaptation of the method in the medical domain. On the other hand, unpaired learning can overcome limitations related to the assumption on noise characteristics, which makes it more feasible for collecting the training data in real-world scenarios. In this paper, we propose a novel image denoising scheme, Interdependent Self-Cooperative Learning (ISCL), that leverages unpaired learning by combining cyclic adversarial learning with self-supervised residual learning. Unlike the existing unpaired image denoising methods relying on matching data distributions in different domains, the two architectures in ISCL, designed for different tasks, complement each other and boost the learning process. To assess the performance of the proposed method, we conducted extensive experiments in various biomedical image degradation scenarios, such as noise caused by physical characteristics of electron microscopy (EM) devices (film and charging noise), and structural noise found in low-dose computer tomography (CT). We demonstrate that the image quality of our method is superior to conventional and current state-of-the-art deep learning-based image denoising methods, including supervised learning.
### SLPC: a VRNN-based approach for stochastic lidar prediction and  completion in autonomous driving
 - **Authors:** George Eskandar, Alexander Braun, Martin Meinke, Karim Armanious, Bin Yang
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Artificial Intelligence (cs.AI)
 - **Arxiv link:** https://arxiv.org/abs/2102.09883
 - **Pdf link:** https://arxiv.org/pdf/2102.09883
 - **Abstract**
 Predicting future 3D LiDAR pointclouds is a challenging task that is useful in many applications in autonomous driving such as trajectory prediction, pose forecasting and decision making. In this work, we propose a new LiDAR prediction framework that is based on generative models namely Variational Recurrent Neural Networks (VRNNs), titled Stochastic LiDAR Prediction and Completion (SLPC). Our algorithm is able to address the limitations of previous video prediction frameworks when dealing with sparse data by spatially inpainting the depth maps in the upcoming frames. Our contributions can thus be summarized as follows: we introduce the new task of predicting and completing depth maps from spatially sparse data, we present a sparse version of VRNNs and an effective self-supervised training method that does not require any labels. Experimental results illustrate the effectiveness of our framework in comparison to the state of the art methods in video prediction.
### Self-Taught Semi-Supervised Anomaly Detection on Upper Limb X-rays
 - **Authors:** Antoine Spahr, Behzad Bozorgtabar, Jean-Philippe Thiran
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2102.09895
 - **Pdf link:** https://arxiv.org/pdf/2102.09895
 - **Abstract**
 Detecting anomalies in musculoskeletal radiographs is of paramount importance for large-scale screening in the radiology workflow. Supervised deep networks take for granted a large number of annotations by radiologists, which is often prohibitively very time-consuming to acquire. Moreover, supervised systems are tailored to closed set scenarios, e.g., trained models suffer from overfitting to previously seen rare anomalies at training. Instead, our approach's rationale is to use task agnostic pretext tasks to leverage unlabeled data based on a cross-sample similarity measure. Besides, we formulate a complex distribution of data from normal class within our framework to avoid a potential bias on the side of anomalies. Through extensive experiments, we show that our method outperforms baselines across unsupervised and self-supervised anomaly detection settings on a real-world medical dataset, the MURA dataset. We also provide rich ablation studies to analyze each training stage's effect and loss terms on the final performance.
### MolCLR: Molecular Contrastive Learning of Representations via Graph  Neural Networks
 - **Authors:** Yuyang Wang, Jianren Wang, Zhonglin Cao, Amir Barati Farimani
 - **Subjects:** Machine Learning (cs.LG); Chemical Physics (physics.chem-ph)
 - **Arxiv link:** https://arxiv.org/abs/2102.10056
 - **Pdf link:** https://arxiv.org/pdf/2102.10056
 - **Abstract**
 Molecular machine learning bears promise for efficient molecule property prediction and drug discovery. However, due to the limited labeled data and the giant chemical space, machine learning models trained via supervised learning perform poorly in generalization. This greatly limits the applications of machine learning methods for molecular design and discovery. In this work, we present MolCLR: Molecular Contrastive Learning of Representations via Graph Neural Networks (GNNs), a self-supervised learning framework for large unlabeled molecule datasets. Specifically, we first build a molecular graph, where each node represents an atom and each edge represents a chemical bond. A GNN is then used to encode the molecule graph. We propose three novel molecule graph augmentations: atom masking, bond deletion, and subgraph removal. A contrastive estimator is utilized to maximize the agreement of different graph augmentations from the same molecule. Experiments show that molecule representations learned by MolCLR can be transferred to multiple downstream molecular property prediction tasks. Our method thus achieves state-of-the-art performance on many challenging datasets. We also prove the efficiency of our proposed molecule graph augmentations on supervised molecular classification tasks.
### Mine Your Own vieW: Self-Supervised Learning Through Across-Sample  Prediction
 - **Authors:** Mehdi Azabou, Mohammad Gheshlaghi Azar, Ran Liu, Chi-Heng Lin, Erik C. Johnson, Kiran Bhaskaran-Nair, Max Dabagia, Keith B. Hengen, William Gray-Roncal, Michal Valko, Eva L. Dyer
 - **Subjects:** Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2102.10106
 - **Pdf link:** https://arxiv.org/pdf/2102.10106
 - **Abstract**
 State-of-the-art methods for self-supervised learning (SSL) build representations by maximizing the similarity between different augmented "views" of a sample. Because these approaches try to match views of the same sample, they can be too myopic and fail to produce meaningful results when augmentations are not sufficiently rich. This motivates the use of the dataset itself to find similar, yet distinct, samples to serve as views for one another. In this paper, we introduce Mine Your Own vieW (MYOW), a new approach for building across-sample prediction into SSL. The idea behind our approach is to actively mine views, finding samples that are close in the representation space of the network, and then predict, from one sample's latent representation, the representation of a nearby sample. In addition to showing the promise of MYOW on standard datasets used in computer vision, we highlight the power of this idea in a novel application in neuroscience where rich augmentations are not already established. When applied to neural datasets, MYOW outperforms other self-supervised approaches in all examples (in some cases by more than 10%), and surpasses the supervised baseline for most datasets. By learning to predict the latent representation of similar samples, we show that it is possible to learn good representations in new domains where augmentations are still limited.
### Error estimates for DeepOnets: A deep learning framework in infinite  dimensions
 - **Authors:** Samuel Lanthaler, Siddhartha Mishra, George Em Karniadakis
 - **Subjects:** Numerical Analysis (math.NA)
 - **Arxiv link:** https://arxiv.org/abs/2102.09618
 - **Pdf link:** https://arxiv.org/pdf/2102.09618
 - **Abstract**
 DeepOnets have recently been proposed as a framework for learning nonlinear operators mapping between infinite dimensional Banach spaces. We analyze DeepOnets and prove estimates on the resulting approximation and generalization errors. In particular, we extend the universal approximation property of DeepOnets to include measurable mappings in non-compact spaces. By a decomposition of the error into encoding, approximation and reconstruction errors, we prove both lower and upper bounds on the total error, relating it to the spectral decay properties of the covariance operators, associated with the underlying measures. We derive almost optimal error bounds with very general affine reconstructors and with random sensor locations as well as bounds on the generalization error, using covering number arguments. We illustrate our general framework with four prototypical examples of nonlinear operators, namely those arising in a nonlinear forced ODE, an elliptic PDE with variable coefficients and nonlinear parabolic and hyperbolic PDEs. In all these examples, we prove that DeepOnets break the curse of dimensionality, thus demonstrating the efficient approximation of infinite-dimensional operators with this machine learning framework.
### Stein variational gradient descent on infinite-dimensional space and  applications to statistical inverse problems
 - **Authors:** Junxiong Jia, Deyu Meng
 - **Subjects:** Numerical Analysis (math.NA)
 - **Arxiv link:** https://arxiv.org/abs/2102.09741
 - **Pdf link:** https://arxiv.org/pdf/2102.09741
 - **Abstract**
 For solving Bayesian inverse problems governed by large-scale forward problems, we present an infinite-dimensional version of the Stein variational gradient descent (iSVGD) method, which has the ability to generate approximate samples from the posteriors efficiently. Specifically, we introduce the concept of the operator-valued kernel and the corresponding function-valued reproducing kernel Hilbert space (RKHS). Through the properties of RKHS, we give an explicit meaning of the infinite-dimensional objects (e.g., the Stein operator) and prove that the infinite-dimensional objects are indeed the limit of finite-dimensional items. Furthermore, by generalizing the change of variables formula, we construct iSVGD with preconditioning operators, yielding more efficient iSVGD. During these generalizations, we introduce a regularity parameter $s\in[0,1]$. Our analysis shows that the intuitive trivial version (i.e., by directly taking finite-dimensional objects as infinite-dimensional items) of iSVGD with preconditioning operators ($s=0$) will yield inaccurate estimates, and the parameter $s$ should be chosen larger than $0$ and smaller than $0.5$. Finally, the proposed algorithms are applied to an inverse problem governed by the Helmholtz equation. Numerical results confirm the correctness of our theoretical findings and demonstrate the potential usefulness of the proposed approach in the posterior sampling of large-scale nonlinear statistical inverse problems.
### Model-Invariant State Abstractions for Model-Based Reinforcement  Learning
 - **Authors:** Manan Tomar, Amy Zhang, Roberto Calandra, Matthew E. Taylor, Joelle Pineau
 - **Subjects:** Machine Learning (cs.LG); Artificial Intelligence (cs.AI); Robotics (cs.RO)
 - **Arxiv link:** https://arxiv.org/abs/2102.09850
 - **Pdf link:** https://arxiv.org/pdf/2102.09850
 - **Abstract**
 Accuracy and generalization of dynamics models is key to the success of model-based reinforcement learning (MBRL). As the complexity of tasks increases, learning dynamics models becomes increasingly sample inefficient for MBRL methods. However, many tasks also exhibit sparsity in the dynamics, i.e., actions have only a local effect on the system dynamics. In this paper, we exploit this property with a causal invariance perspective in the single-task setting, introducing a new type of state abstraction called \textit{model-invariance}. Unlike previous forms of state abstractions, a model-invariance state abstraction leverages causal sparsity over state variables. This allows for generalization to novel combinations of unseen values of state variables, something that non-factored forms of state abstractions cannot do. We prove that an optimal policy can be learned over this model-invariance state abstraction. Next, we propose a practical method to approximately learn a model-invariant representation for complex domains. We validate our approach by showing improved modeling performance over standard maximum likelihood approaches on challenging tasks, such as the MuJoCo-based Humanoid. Furthermore, within the MBRL setting we show strong performance gains w.r.t. sample efficiency across a host of other continuous control tasks.
### Gerrymandering on graphs: Computational complexity and parameterized  algorithms
 - **Authors:** Sushmita Gupta, Pallavi Jain, Fahad, Panolan, Sanjukta Roy, Saket Saurabh
 - **Subjects:** Data Structures and Algorithms (cs.DS); Computer Science and Game Theory (cs.GT)
 - **Arxiv link:** https://arxiv.org/abs/2102.09889
 - **Pdf link:** https://arxiv.org/pdf/2102.09889
 - **Abstract**
 Partitioning a region into districts to favor a particular candidate or a party is commonly known as gerrymandering. In this paper, we investigate the gerrymandering problem in graph theoretic setting as proposed by Cohen-Zemach et al. [AAMAS 2018]. Our contributions in this article are two-fold, conceptual and computational. We first resolve the open question posed by Ito et al. [AAMAS 2019] about the computational complexity of the problem when the input graph is a path. Next, we propose a generalization of their model, where the input consists of a graph on $n$ vertices representing the set of voters, a set of $m$ candidates $\mathcal{C}$, a weight function $w_v: \mathcal{C}\rightarrow {\mathbb Z}^+$ for each voter $v\in V(G)$ representing the preference of the voter over the candidates, a distinguished candidate $p\in \mathcal{C}$, and a positive integer $k$. The objective is to decide if one can partition the vertex set into $k$ pairwise disjoint connected sets (districts) s.t $p$ wins more districts than any other candidate. The problem is known to be NPC even if $k=2$, $m=2$, and $G$ is either a complete bipartite graph (in fact $K_{2,n}$) or a complete graph. This means that in search for FPT algorithms we need to either focus on the parameter $n$, or subclasses of forest. Circumventing these intractable results, we give a deterministic and a randomized algorithms for the problem on paths running in times $2.619^{k}(n+m)^{O(1)}$ and $2^{k}(n+m)^{O(1)}$, respectively. Additionally, we prove that the problem on general graphs is solvable in time $2^n (n+m)^{O(1)}$. Our algorithmic results use sophisticated technical tools such as representative set family and Fast Fourier transform based polynomial multiplication, and their (possibly first) application to problems arising in social choice theory and/or game theory may be of independent interest to the community.
### Implicit Regularization in Tensor Factorization
 - **Authors:** Noam Razin, Asaf Maman, Nadav Cohen
 - **Subjects:** Machine Learning (cs.LG); Artificial Intelligence (cs.AI); Neural and Evolutionary Computing (cs.NE); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2102.09972
 - **Pdf link:** https://arxiv.org/pdf/2102.09972
 - **Abstract**
 Implicit regularization in deep learning is perceived as a tendency of gradient-based optimization to fit training data with predictors of minimal "complexity." The fact that only some types of data give rise to generalization is understood to result from them being especially amenable to fitting with low complexity predictors. A major challenge towards formalizing this intuition is to define complexity measures that are quantitative yet capture the essence of data that admits generalization. With an eye towards this challenge, we analyze the implicit regularization in tensor factorization, equivalent to a certain non-linear neural network. We characterize the dynamics that gradient descent induces on the factorization, and establish a bias towards low tensor rank, in compliance with existing empirical evidence. Then, motivated by tensor rank capturing implicit regularization of a non-linear neural network, we empirically explore it as a measure of complexity, and find that it stays extremely low when fitting standard datasets. This leads us to believe that tensor rank may pave way to explaining both implicit regularization of neural networks, and the properties of real-world data translating it to generalization.
### MolCLR: Molecular Contrastive Learning of Representations via Graph  Neural Networks
 - **Authors:** Yuyang Wang, Jianren Wang, Zhonglin Cao, Amir Barati Farimani
 - **Subjects:** Machine Learning (cs.LG); Chemical Physics (physics.chem-ph)
 - **Arxiv link:** https://arxiv.org/abs/2102.10056
 - **Pdf link:** https://arxiv.org/pdf/2102.10056
 - **Abstract**
 Molecular machine learning bears promise for efficient molecule property prediction and drug discovery. However, due to the limited labeled data and the giant chemical space, machine learning models trained via supervised learning perform poorly in generalization. This greatly limits the applications of machine learning methods for molecular design and discovery. In this work, we present MolCLR: Molecular Contrastive Learning of Representations via Graph Neural Networks (GNNs), a self-supervised learning framework for large unlabeled molecule datasets. Specifically, we first build a molecular graph, where each node represents an atom and each edge represents a chemical bond. A GNN is then used to encode the molecule graph. We propose three novel molecule graph augmentations: atom masking, bond deletion, and subgraph removal. A contrastive estimator is utilized to maximize the agreement of different graph augmentations from the same molecule. Experiments show that molecule representations learned by MolCLR can be transferred to multiple downstream molecular property prediction tasks. Our method thus achieves state-of-the-art performance on many challenging datasets. We also prove the efficiency of our proposed molecule graph augmentations on supervised molecular classification tasks.
### Interpretable Stability Bounds for Spectral Graph Filters
 - **Authors:** Henry Kenlay, Dorina Thanou, Xiaowen Dong
 - **Subjects:** Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2102.09587
 - **Pdf link:** https://arxiv.org/pdf/2102.09587
 - **Abstract**
 Graph-structured data arise in a variety of real-world context ranging from sensor and transportation to biological and social networks. As a ubiquitous tool to process graph-structured data, spectral graph filters have been used to solve common tasks such as denoising and anomaly detection, as well as design deep learning architectures such as graph neural networks. Despite being an important tool, there is a lack of theoretical understanding of the stability properties of spectral graph filters, which are important for designing robust machine learning models. In this paper, we study filter stability and provide a novel and interpretable upper bound on the change of filter output, where the bound is expressed in terms of the endpoint degrees of the deleted and newly added edges, as well as the spatial proximity of those edges. This upper bound allows us to reason, in terms of structural properties of the graph, when a spectral graph filter will be stable. We further perform extensive experiments to verify intuition that can be gained from the bound.
### Improving DeepFake Detection Using Dynamic Face Augmentation
 - **Authors:** Sowmen Das, Arup Datta, Md. Saiful Islam, Md. Ruhul Amin
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Artificial Intelligence (cs.AI); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2102.09603
 - **Pdf link:** https://arxiv.org/pdf/2102.09603
 - **Abstract**
 The creation of altered and manipulated faces has become more common due to the improvement of DeepFake generation methods. Simultaneously, we have seen detection models' development for differentiating between a manipulated and original face from image or video content. We have observed that most publicly available DeepFake detection datasets have limited variations, where a single face is used in many videos, resulting in an oversampled training dataset. Due to this, deep neural networks tend to overfit to the facial features instead of learning to detect manipulation features of DeepFake content. As a result, most detection architectures perform poorly when tested on unseen data. In this paper, we provide a quantitative analysis to investigate this problem and present a solution to prevent model overfitting due to the high volume of samples generated from a small number of actors. We introduce Face-Cutout, a data augmentation method for training Convolutional Neural Networks (CNN), to improve DeepFake detection. In this method, training images with various occlusions are dynamically generated using face landmark information irrespective of orientation. Unlike other general-purpose augmentation methods, it focuses on the facial information that is crucial for DeepFake detection. Our method achieves a reduction in LogLoss of 15.2% to 35.3% on different datasets, compared to other occlusion-based augmentation techniques. We show that Face-Cutout can be easily integrated with any CNN-based recognition model and improve detection performance.
### Modelling Paralinguistic Properties in Conversational Speech to Detect  Bipolar Disorder and Borderline Personality Disorder
 - **Authors:** Bo Wang, Yue Wu, Nemanja Vaci, Maria Liakata, Terry Lyons, Kate E A Saunders
 - **Subjects:** Machine Learning (cs.LG); Sound (cs.SD); Audio and Speech Processing (eess.AS)
 - **Arxiv link:** https://arxiv.org/abs/2102.09607
 - **Pdf link:** https://arxiv.org/pdf/2102.09607
 - **Abstract**
 Bipolar disorder (BD) and borderline personality disorder (BPD) are two chronic mental health conditions that clinicians find challenging to distinguish based on clinical interviews, due to their overlapping symptoms. In this work, we investigate the automatic detection of these two conditions by modelling both verbal and non-verbal cues in a set of interviews. We propose a new approach of modelling short-term features with visibility-signature transform, and compare it with widely used high-level statistical functions. We demonstrate the superior performance of our proposed signature-based model. Furthermore, we show the role of different sets of features in characterising BD and BPD.
### Detecting Communities in Gossip Model with Stubborn Agents
 - **Authors:** Yu Xing, Xingkang He, Haitao Fang, Karl H. Johansson
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2102.09683
 - **Pdf link:** https://arxiv.org/pdf/2102.09683
 - **Abstract**
 We consider a community detection problem in a gossip model, where agents randomly interact pairwise, with stubborn agents never changing their states. It is assumed that the agents can be divided into two communities based on their interaction probability with others. Such a model can illustrate how disagreement and opinion fluctuation arise in a social network. The considered problem is twofold: to infer which community each agent belongs to, and to estimate interaction probabilities between agents, by only observing their state evolution. First, stability and limit theorems of the model are derived. An integrated detection and estimation algorithm is then proposed to infer the two communities and to estimate the interaction probabilities, based on agent states. It is verified that the community detector of the algorithm converges in finite time, and the interaction estimator converges almost surely. In addition, non-asymptotic property is obtained for the former, and convergence rate is analyzed for the latter. Simulations are presented to illustrate the performance of the proposed algorithm.
### Tight Trade-off in Contention Resolution without Collision Detection
 - **Authors:** Haimin Chen, Yonggang Jiang, Chaodong Zheng
 - **Subjects:** Distributed, Parallel, and Cluster Computing (cs.DC)
 - **Arxiv link:** https://arxiv.org/abs/2102.09716
 - **Pdf link:** https://arxiv.org/pdf/2102.09716
 - **Abstract**
 In this paper, we consider contention resolution on a multiple-access communication channel. In this problem, a set of nodes arrive over time, each with a message it intends to send. In each time slot, each node may attempt to broadcast its message or remain idle. If a single node broadcasts in a slot, the message is received by all nodes; otherwise, if multiple nodes broadcast simultaneously, a collision occurs and none succeeds. If collision detection is available, nodes can differentiate collision and silence (i.e., no nodes broadcast). Performance of contention resolution algorithms is often measured by throughput -- the number of successful transmissions within a period of time; whereas robustness is often measured by jamming resistance -- a jammed slot always generates a collision. Previous work has shown, with collision detection, optimal constant throughput can be attained, even if a constant fraction of all slots are jammed. The situation when collision detection is not available, however, remains unclear. In a recent breakthrough paper [Bender et al., STOC '20], a crucial case is resolved: constant throughput is possible without collision detection, but only if there is no jamming. Nonetheless, the exact trade-off between the best possible throughput and the severity of jamming remains unknown. In this paper, we address this open question. Specifically, for any level of jamming ranging from none to constant fraction, we prove an upper bound on the best possible throughput, along with an algorithm attaining that bound. An immediate and interesting implication of our result is, when a constant fraction of all slots are jammed, which is the worst-case scenario, there still exists an algorithm achieving a decent throughput: $\Theta(t/\log{t})$ messages could be successfully transmitted within $t$ slots.
### Serial-parallel Multi-Scale Feature Fusion for Anatomy-Oriented Hand  Joint Detection
 - **Authors:** Bin Li, Hong Fu, Ruimin Li, Wendi Wang
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2102.09757
 - **Pdf link:** https://arxiv.org/pdf/2102.09757
 - **Abstract**
 Accurate hand joints detection from images is a fundamental topic which is essential for many applications in computer vision and human computer interaction. This paper presents a two stage network for hand joints detection from single unmarked image by using serial-parallel multi-scale feature fusion. In stage I, the hand regions are located by a pre-trained network, and the features of each detected hand region are extracted by a shallow spatial hand features representation module. The extracted hand features are then fed into stage II, which consists of serially connected feature extraction modules with similar structures, called "multi-scale feature fusion" (MSFF). A MSFF contains parallel multi-scale feature extraction branches, which generate initial hand joint heatmaps. The initial heatmaps are then mutually reinforced by the anatomic relationship between hand joints. The experimental results on five hand joints datasets show that the proposed network overperforms the state-of-the-art methods.
### Applications of deep learning in traffic congestion alleviation: A  survey
 - **Authors:** Nishant Kumar, Martin Raubal
 - **Subjects:** Machine Learning (cs.LG); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2102.09759
 - **Pdf link:** https://arxiv.org/pdf/2102.09759
 - **Abstract**
 Prediction tasks related to congestion are targeted at improving the level of service of the transportation network. With increasing access to larger datasets of higher resolution, the relevance of deep learning in such prediction tasks, is increasing. Several comprehensive survey papers in recent years have summarised the deep learning applications in the transportation domain. However, the system dynamics of the transportation network vary greatly between the non-congested state and the congested state -- thereby necessitating the need for a clear understanding of the challenges specific to congestion prediction. In this survey, we present the current state of deep learning applications in the tasks related to detection, prediction and propagation of congestion. Recurrent and non-recurrent congestion are discussed separately. Our survey leads us to uncover inherent challenges and gaps in the current state of research. Finally, we present some suggestions for future research directions as answers to the identified challenges.
### Characterizing and Mitigating Self-Admitted Build Debt
 - **Authors:** Tao Xiao, Dong Wang, Shane McIntosh, Hideaki Hata, Raula Gaikovina Kula, Takashi Ishio, Kenichi Matsumoto
 - **Subjects:** Software Engineering (cs.SE)
 - **Arxiv link:** https://arxiv.org/abs/2102.09775
 - **Pdf link:** https://arxiv.org/pdf/2102.09775
 - **Abstract**
 Technical Debt is a metaphor used to describe the situation in which long-term code quality is traded for short-term goals in software projects. In recent years, the concept of self-admitted technical debt (SATD) was proposed, which focuses on debt that is intentionally introduced and described by developers. Although prior work has made important observations about admitted technical debt in source code, little is known about SATD in build systems. In this paper, we coin the term Self-Admitted Build Debt (SABD) and through a qualitative analysis of 500 SABD comments in the Maven build system of 300 projects, we characterize SABD by location and rationale (reason and purpose). Our results show that limitations in tools and libraries, and complexities of dependency management are the most frequent causes, accounting for 49% and 23% of the comments. We also find that developers often document SABD as issues to be fixed later. To automate the detection of SABD rationale, we train classifiers to label comments according to the surrounding document content. The classifier performance is promising, achieving an F1-score of 0.67-0.75. Finally, within 16 identified 'ready-to-be-addressed' SABD instances, the three SABD submitted by pull requests and the five SABD submitted by issue reports were resolved after developers were made aware. Our work presents the first step towards understanding technical debt in build systems and opens up avenues for future work, such as tool support to track and manage SABD backlogs.
### Defense against flooding attacks using probabilistic thresholds in the  internet of things ecosystem
 - **Authors:** Seyed Meysam Zarei, Reza Fotohi
 - **Subjects:** Cryptography and Security (cs.CR); Networking and Internet Architecture (cs.NI)
 - **Arxiv link:** https://arxiv.org/abs/2102.09805
 - **Pdf link:** https://arxiv.org/pdf/2102.09805
 - **Abstract**
 The Internet of Things (IoT) ecosystem allows communication between billions of devices worldwide that are collecting data autonomously. The vast amount of data generated by these devices must be controlled totally securely. The centralized solutions are not capable of responding to these concerns due to security challenges problems. Thus, the Average Packet Transmission RREQ (APT-RREQ) as an effective solution, has been employed to overcome these concerns to allow for entirely secure communication between devices. In this paper, an approach called LSFA-IoT is proposed that protects the AODV routing protocol as well as the IoT network against flooding. The proposed method is divided into two main phases; The first phase includes a physical layer intrusion and attack detection system used to detect attacks, and the second phase involves detecting incorrect events through APT-RREQ messages. The simulation results indicated the superiority of the proposed method in terms of False Positive Rate (FPR), False Negative Rate (FPR), Detection Rate (DR), and Packet Delivery Rate (PDR) compared to REATO and IRAD. Also, the simulation results show how the proposed approach can significantly increase the security of each thing and network security.
### Training cascaded networks for speeded decisions using a  temporal-difference loss
 - **Authors:** Michael L. Iuzzolino, Michael C. Mozer, Samy Bengio
 - **Subjects:** Machine Learning (cs.LG); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2102.09808
 - **Pdf link:** https://arxiv.org/pdf/2102.09808
 - **Abstract**
 Although deep feedforward neural networks share some characteristics with the primate visual system, a key distinction is their dynamics. Deep nets typically operate in sequential stages wherein each layer fully completes its computation before processing begins in subsequent layers. In contrast, biological systems have cascaded dynamics: information propagates from neurons at all layers in parallel but transmission is gradual over time. In our work, we construct a cascaded ResNet by introducing a propagation delay into each residual block and updating all layers in parallel in a stateful manner. Because information transmitted through skip connections avoids delays, the functional depth of the architecture increases over time and yields a trade off between processing speed and accuracy. We introduce a temporal-difference (TD) training loss that achieves a strictly superior speed accuracy profile over standard losses. The CascadedTD model has intriguing properties, including: typical instances are classified more rapidly than atypical instances; CascadedTD is more robust to both persistent and transient noise than is a conventional ResNet; and the time-varying output trace of CascadedTD provides a signal that can be used by `meta-cognitive' models for OOD detection and to determine when to terminate processing.
### Introducing an experimental distortion-tolerant speech encryption scheme  for secure voice communication
 - **Authors:** Piotr Krasnowski, Jerome Lebrun, Bruno Martin
 - **Subjects:** Cryptography and Security (cs.CR)
 - **Arxiv link:** https://arxiv.org/abs/2102.09809
 - **Pdf link:** https://arxiv.org/pdf/2102.09809
 - **Abstract**
 The current increasing need for privacy-preserving voice communications is leading to new ideas for securing voice transmission. This paper refers to a relatively new concept of sending encrypted speech as pseudo-speech in the audio domain over digital voice communication infrastructures, like 3G cellular network and VoIP. This work presents a novel distortion-tolerant speech encryption scheme for secure voice communications over voice channels that combines the robustness of analog speech scrambling and elevated security offered by digital ciphers like AES-CTR. The system scrambles vocal parameters of a speech signal (loudness, pitch, timbre) using distance-preserving pseudo-random translations and rotations on a hypersphere of parameters. Next, scrambled parameters are encoded to a pseudo-speech signal adapted to transmission over digital voice channels equipped with voice activity detection. Upon reception of this pseudo-speech signal, the legitimate receiver restores distorted copies of the initial vocal parameters. Despite some deciphering errors, an integrated neural-based vocoder based on the LPCNet architecture reconstructs an intelligible speech. The experimental implementation of this speech encryption scheme has been tested by simulations and sending an encrypted signal over FaceTime between two iPhones 6 connected to the same WiFi network. Moreover, speech excerpts restored from encrypted signals were evaluated by a speech quality assessment on a group of about 40 participants. The experiments demonstrated that the proposed scheme produces intelligible speech with a gracefully progressive quality degradation depending on the channel noise. Finally, the preliminary computational analysis suggested that the presented setting may operate on high-end portable devices in nearly real-time.
### Self-Taught Semi-Supervised Anomaly Detection on Upper Limb X-rays
 - **Authors:** Antoine Spahr, Behzad Bozorgtabar, Jean-Philippe Thiran
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2102.09895
 - **Pdf link:** https://arxiv.org/pdf/2102.09895
 - **Abstract**
 Detecting anomalies in musculoskeletal radiographs is of paramount importance for large-scale screening in the radiology workflow. Supervised deep networks take for granted a large number of annotations by radiologists, which is often prohibitively very time-consuming to acquire. Moreover, supervised systems are tailored to closed set scenarios, e.g., trained models suffer from overfitting to previously seen rare anomalies at training. Instead, our approach's rationale is to use task agnostic pretext tasks to leverage unlabeled data based on a cross-sample similarity measure. Besides, we formulate a complex distribution of data from normal class within our framework to avoid a potential bias on the side of anomalies. Through extensive experiments, we show that our method outperforms baselines across unsupervised and self-supervised anomaly detection settings on a real-world medical dataset, the MURA dataset. We also provide rich ablation studies to analyze each training stage's effect and loss terms on the final performance.
### DemSelf, a Mobile App for Self-Administered Touch-Based Cognitive  Screening: Participatory Design With Stakeholders
 - **Authors:** Martin Burghart, Julie L. O'Sullivan, Robert Spang, Jan-Niklas Voigt-Antons
 - **Subjects:** Human-Computer Interaction (cs.HC)
 - **Arxiv link:** https://arxiv.org/abs/2102.09940
 - **Pdf link:** https://arxiv.org/pdf/2102.09940
 - **Abstract**
 Early detection of mild cognitive impairment and dementia is vital as many therapeutic interventions are particularly effective at an early stage. A self-administered touch-based cognitive screening instrument, called DemSelf, was developed by adapting an examiner-administered paper-based instrument, the Quick Mild Cognitive Impairment (Qmci) screen. We conducted five semi-structured expert interviews including a think-aloud phase to evaluate usability problems. The extent to which the characteristics of the original subtests change by the adaption, as well as the conditions and appropriate context for practical application, were also in question. The participants had expertise in the domain of usability and human-machine interaction and/or in the domain of dementia and neuropsychological assessment. Participants identified usability issues in all components of the DemSelf prototype. For example, confirmation of answers was not consistent across subtests. Answers were sometimes logged directly when a button is tapped and cannot be corrected. This can lead to frustration and bias in test results, especially for people with vision or motor impairments. The direct adoption of time limits from the original paper-based instrument or the simultaneous verbal and textual item presentation also caused usability problems. DemSelf is a different test than Qmci and needs to be re-validated. Visual recognition instead of a free verbal recall is one of the main differences. Reading skill level seems to be an important confounding variable. Participants would generally prefer if the test is conducted in a medical office rather than at a patient's home so that someone is present for support and the result can be discussed directly.
### Towards Emotion Recognition in Hindi-English Code-Mixed Data: A  Transformer Based Approach
 - **Authors:** Anshul Wadhawan, Akshita Aggarwal
 - **Subjects:** Computation and Language (cs.CL)
 - **Arxiv link:** https://arxiv.org/abs/2102.09943
 - **Pdf link:** https://arxiv.org/pdf/2102.09943
 - **Abstract**
 In the last few years, emotion detection in social-media text has become a popular problem due to its wide ranging application in better understanding the consumers, in psychology, in aiding human interaction with computers, designing smart systems etc. Because of the availability of huge amounts of data from social-media, which is regularly used for expressing sentiments and opinions, this problem has garnered great attention. In this paper, we present a Hinglish dataset labelled for emotion detection. We highlight a deep learning based approach for detecting emotions in Hindi-English code mixed tweets, using bilingual word embeddings derived from FastText and Word2Vec approaches, as well as transformer based models. We experiment with various deep learning models, including CNNs, LSTMs, Bi-directional LSTMs (with and without attention), along with transformers like BERT, RoBERTa, and ALBERT. The transformer based BERT model outperforms all other models giving the best performance with an accuracy of 71.43%.
### A flow-based IDS using Machine Learning in eBPF
 - **Authors:** Maximilian Bachl, Joachim Fabini, Tanja Zseby
 - **Subjects:** Cryptography and Security (cs.CR); Machine Learning (cs.LG); Networking and Internet Architecture (cs.NI); Operating Systems (cs.OS)
 - **Arxiv link:** https://arxiv.org/abs/2102.09980
 - **Pdf link:** https://arxiv.org/pdf/2102.09980
 - **Abstract**
 eBPF is a new technology which allows dynamically loading pieces of code into the Linux kernel. It can greatly speed up networking since it enables the kernel to process certain packets without the involvement of a userspace program. So far eBPF has been used for simple packet filtering applications such as firewalls or Denial of Service protection. We show that it is possible to develop a flow based network intrusion detection system based on machine learning entirely in eBPF. Our solution uses a decision tree and decides for each packet whether it is malicious or not, considering the entire previous context of the network flow. We achieve a performance increase of over 20\% compared to the same solution implemented as a userspace program.
### Effective and Efficient Vote Attack on Capsule Networks
 - **Authors:** Jindong Gu, Baoyuan Wu, Volker Tresp
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2102.10055
 - **Pdf link:** https://arxiv.org/pdf/2102.10055
 - **Abstract**
 Standard Convolutional Neural Networks (CNNs) can be easily fooled by images with small quasi-imperceptible artificial perturbations. As alternatives to CNNs, the recently proposed Capsule Networks (CapsNets) are shown to be more robust to white-box attacks than CNNs under popular attack protocols. Besides, the class-conditional reconstruction part of CapsNets is also used to detect adversarial examples. In this work, we investigate the adversarial robustness of CapsNets, especially how the inner workings of CapsNets change when the output capsules are attacked. The first observation is that adversarial examples misled CapsNets by manipulating the votes from primary capsules. Another observation is the high computational cost, when we directly apply multi-step attack methods designed for CNNs to attack CapsNets, due to the computationally expensive routing mechanism. Motivated by these two observations, we propose a novel vote attack where we attack votes of CapsNets directly. Our vote attack is not only effective but also efficient by circumventing the routing process. Furthermore, we integrate our vote attack into the detection-aware attack paradigm, which can successfully bypass the class-conditional reconstruction based detection method. Extensive experiments demonstrate the superior attack performance of our vote attack on CapsNets.
## Keyword: line segment
There is no result 
## Keyword: text to image
There is no result
